Maskbook/Supporting Files/Coordinator.swift

```diff
@@ -87,6 +87,7 @@ class Coordinator {
         case selectItemViewController(viewModel: SelectItemViewModelProtocol)
         case selectActionViewController(viewModel: SelectActionViewModelProtocol)
         case alertController(alertController: AlertController)
+        case viewController(viewController: UIViewController)
```

Comment:
Why add this case? A object like `ViewModel`, `UIViewController` should not be passed around via `Coordinator pattern`.

Reply:
Do you have a simple way to present a viewController in `ViewModel`

Reply:
1. Do the presenting in the viewController that holds the viewModel by callback, combine steam or delegate pattern, this will decouple the business logic and navigation logic.
2. Add a callback in the coordinator case will be the simplest solution for now, may be later we should refactor the coordinator we are using.